spectrocloud/data_source_appliances.go

```diff
@@ -0,0 +1,73 @@
+package spectrocloud
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/spectrocloud/hapi/models"
+	"github.com/spectrocloud/terraform-provider-spectrocloud/pkg/client"
+	"reflect"
+)
+
+type MyMap map[string]string
+
+func dataSourceAppliances() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourcesApplianceRead,
```

Comment:
```suggestion
		ReadContext: dataSourcesApplianceRead,
		Description: "Provides details about a specific appliance.",
```

There is a high-level description field that we need to add to the resource. The example below is for API Gatewayv2. The message "Manages an Amazon API Gateway Version 2 API." comes from the resource level description. 

![CleanShot 2022-12-01 at 12 25 01](https://user-images.githubusercontent.com/29551334/205141788-18adf421-2aea-4368-aad8-ad734e3aa6ac.png)